hieroplatform/src/main/java/org/hiero/sketch/table/SemiExplicitConverter.java

```diff
@@ -0,0 +1,18 @@
+package org.hiero.sketch.table;
+
+public class SemiExplicitConverter extends BaseExplicitConverter {
+    final double defaultValue;
```

Comment:
How about we make defaultValue be the constant -1?
One less thing to keep track of.